app/views/StartPageView.scala.html

```diff
@@ -0,0 +1,189 @@
+
+    @bullets(Seq(
+        "the EPU number, entry number and entry date",
+        "all item numbers",
+        "the type of duty or tax that was paid",
+        "the amount of tax or duty that was paid and the amount that should have been paid",
+        "the original and amended values of any information that was wrong in the original import declaration (for example, commodity codes or invoice values)").map(Html(_)))
+
+    @p(Html("You must not include import declarations where some have an entry acceptance date before and some after 1 January 2021. If the entries cross this date you will need to separate them and make two disclosures.\n"))
+
+    @h3("Deferment account details")
+
+    @p(Html("If you are paying by deferment account, you will need to supply the deferment account number (DAN). Where it is not your own deferment account you will need to have one of the following:"))
+
+    @bullets(Seq(
+        "standing authority as indicated on CHIEF",
+        "a document provided by the owner of the deferment account giving you permission to use it. This must be dated after the import declaration was made.").map(Html(_)))
+
+    @h3("Disclosing underpaid import tax or duty on behalf of an importer")
+
+    @p(Html("If you are acting on behalf of an importer you will need to provide:"))
+
+    @bullets(Seq(
+        "the importer's EORI number (if they have one)",
+        "the importer's trading name",
+        "whether they are VAT registered",
+        "their address so that we can send them a copy of the demand note").map(Html(_)))
+
+    @h2("How to disclose", Some("4"))
+
+    @p(Html("You'll need the Government Gateway user ID and password you used to register your EORI number. You may also be asked to subscribe or link your EORI number to your Government Gateway user ID. You will only be asked to do this once."))
+
+    @p(Html("You need to complete the disclosure in one go. You cannot save your answers and return later."))
+
+    <a href="@controllers.routes.IndexController.onPageLoad()" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+        Start now
+        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+        </svg></a>
+
+    @h2("After you have disclosed", Some("5"))
+
+    @p(Html("We will check the information you have provided and if we agree with the calculations we will send a C18 Post Clearance Demand Note in the post within 14 days."))
+
+    @p(Html("The demand note will confirm what is owed to HMRC and, if you have not paid by deferment, will tell you how to pay. Payment must be made within 10 days of receiving the demand note otherwise interest will become due."))
```

Comment:
Can you just check this content is still correct with Nikki as we removed it from our Confirmation page due to it being no longer correct as interest isn't charged.